vms/shift/services.py

```diff
@@ -331,10 +331,11 @@ def get_shifts_with_open_slots_for_volunteer(j_id, v_id):
     """
     shift_list_by_date = get_shifts_ordered_by_date(j_id)
     shift_list = []
+    today_date= datetime.date.today()
 

```

Comment:
As we have already imported datetime in this file, so this statement will work without dotting it as well. 

Can you please remove it?